packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -1,137 +1,48 @@
+/* tslint:disable:no-unused-variable */
 import * as React from 'react';
-import {
-  BaseComponent,
-  css,
-  assign,
-  getId,
-  getNativeProps,
-  buttonProperties,
-  anchorProperties
-} from '../../Utilities';
-import { IButtonProps, IButton, ButtonType } from './Button.Props';
-import './ButtonCore/ButtonCore.scss';
-import './CommandButton/CommandButton.scss';
-import './CompoundButton/CompoundButton.scss';
-import './DefaultButton/DefaultButton.scss';
-import './IconButton/IconButton.scss';
-import './PrimaryButton/PrimaryButton.scss';
+/* tslint:enable:no-unused-variable */
 
-export interface IButtonState {
-  labelId?: string;
-  descriptionId?: string;
-  ariaDescriptionId?: string;
-}
-
-export class Button extends BaseComponent<IButtonProps, IButtonState> implements IButton {
-  public static defaultProps: IButtonProps = {
-    buttonType: ButtonType.default
-  };
-  private _buttonElement: HTMLButtonElement;
-
-  constructor(props: IButtonProps) {
-    super(props, { 'rootProps': null });
-
-    this.state = {
-      labelId: getId('Button'),
-      descriptionId: getId('Button'),
-      ariaDescriptionId: getId('Button')
-    };
-  }
+import { BaseComponent } from '../../Utilities';
+import { ButtonType, IButton, IButtonProps } from './Button.Props';
+import { BaseButton } from './BaseButton';
 
-  public render(): JSX.Element {
-    let { buttonType, children, icon, description, ariaLabel, ariaDescription, href, disabled, onClick } = this.props;
-    let { labelId, descriptionId, ariaDescriptionId } = this.state;
+import { DefaultButton } from './DefaultButton/DefaultButton';
+import { CommandButton } from './CommandButton/CommandButton';
+import { CompoundButton } from './CompoundButton/CompoundButton';
+import { IconButton } from './IconButton/IconButton';
+import { PrimaryButton } from './PrimaryButton/PrimaryButton';
 
```

Comment:
Are you removing some of the css associated with this button? More importantly does the css and look of this button change at all? Just might be soemthing that could break it if people are adjusting for it.